common/clickhouse-srv/src/binary/encoder.rs

```diff
@@ -12,8 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use common_io::prelude::Marshal;
+
 use crate::binary;
-use crate::types::Marshal;
 use crate::types::StatBuffer;
```

Comment:
The `StatBuffer` can also be removed?

Reply:
I will check it later.
The cargo clippy not warning me that.

Reply:
Checked.
The clickhouse-srv StatBuffer is not same as io/StatBuffer, it has a `sql_type` in his trait.
Shall we merge them in this patch?